### DISCUSSION TITLE
Using wayland instead of x11

### DISCUSSION THREAD

**Discussion by frustak** [microsoft/wslg#940] [Q&A]
I couldn't find anyway to do it in WSL, How can I make GUI apps to use Wayland instead of X11?

**Comment by MWatter** [comment]
No answers. Too bad. I have the same issue. I know that Wayland is technically available because if I launch firefox with `MOZ_ENABLE_WAYLAND=1 firefox` it looks great, but just `firefox`, and it's clearly using X11 and not looking so great. 

How do I get my applications to launch with Wayland? It's fine if I have to create a script to launch each one; I only have a few that I need to keep open.

**Reply by MWatter** [reply]
Ok, so I guess GTK based apps use Wayland by default? I don't use gedit or nautilus but when I did try them, sure enough, they use wayland. What about others? I use Intellij IDEA, Konsole, and Chrome. Is there something special for non-GTK apps?

**Comment by MamiyaOtaru** [comment]
export XDG_SESSION_TYPE=wayland
export QT_QPA_PLATFORM=wayland
will catch some more apps

**Reply by kosl** [reply]
When I am runing Qt6 application (e.g. designer) with QT_QPA_PLATFORM=wayland then I don't get window manager around it (as is the case when using QT_QPA_PLATFORM=xcb)  to move or close the app. Is there a way to get Weston catching Wayland windows?
![image](https://github.com/microsoft/wslg/assets/4527143/b8fbfe7a-907c-4e95-8620-37d54b779d6a)
